example/build.gradle

```diff
@@ -12,10 +12,11 @@ dependencies {
     compile 'com.android.support:support-v4:' + android_support_version
     compile 'com.android.support:recyclerview-v7:' + android_support_version
     /* Needed for RxAndroid */
+    /* Needed for Rx Bindings on views */
     compile 'io.reactivex:rxandroid:1.2.1'
     compile 'io.reactivex:rxjava:1.1.6'
-    /* Needed for Rx Bindings on views */
     compile 'com.jakewharton.rxbinding:rxbinding:0.4.0'
+    compile 'com.android.support.constraint:constraint-layout:1.0.0-beta3'
```

Comment:
How does this keep getting auto-added? Going to remove, because it causes a travis failure. Also, I don't use this.